### DISCUSSION TITLE
monaco-languageclient does not support the latest monaco-editor 0.37.1

### DISCUSSION THREAD

**Discussion by abdalla-rko** [TypeFox/monaco-languageclient#486] [Q&A] [answered]
Hi,

I'm not able to load the latest monaco-editor with the current monaco-languageclient. I'm getting a blank screen. Any ideas or insights that could help resolve this would be greatly appreciated.

Thanks

**Comment by CGNonofr** [comment, marked as answer]
this lib is not yet compatible with monaco 0.37, it'll come soon

**Reply by kaisalmen** [reply]
Preview 5.1.0-next.0 using monaco-vscode-api@1.78.0 and monaco-editor@0.37.1is now available:
https://github.com/TypeFox/monaco-languageclient/pull/485#issuecomment-1511099414
